scripts/run-litestream.sh

```diff
@@ -0,0 +1,32 @@
+# only set these if they arent set in the environment
+if [ -z "$COLLECTOR_LITESTREAM_URL" ]; then
+    export COLLECTOR_LITESTREAM_URL="gcs://ldrpc-seed-data/itn4/collector"
+fi
+if [ -z "$SERVICE_VALIDATOR_LITESTREAM_URL" ]; then
+    export SERVICE_VALIDATOR_LITESTREAM_URL="gcs://ldrpc-seed-data/itn4/shardeum"
+fi
+
+# convert base64 to json
+cat /home/node/dummy.json | base64 --decode > /home/node/dummy_creds.json
+export GOOGLE_APPLICATION_CREDENTIALS="/home/node/dummy_creds.json"
+
+# if we have a collector database already dont litestream restore it
+if [ ! -f "$COLLECTOR_DB_PATH" ]; then
+    echo "Restoring collector database at $COLLECTOR_DB_PATH"
+    litestream restore -o "$COLLECTOR_DB_PATH" "$COLLECTOR_LITESTREAM_URL"
+    # Repair litestream if tmp file exists and the db doesnt exist
+    if [ -f "$COLLECTOR_DB_PATH.tmp" ] && [ ! -f "$COLLECTOR_DB_PATH" ]; then
```

Comment:
@shanejonas IMO we shouldn't be doing this. We can't be sure if .tmp file is correct and litestream  deliberately didn't rename it. We have already seen sqlite_corrupt error with wget option. so I would expect litestream to restore data correctly or its a failed restore and we shouldn't do anything to touch those db files